pages/price-feeds/use-real-time-data/evm.mdx

```diff
@@ -70,21 +48,38 @@ import "@pythnetwork/pyth-sdk-solidity/PythStructs.sol";
 contract SomeContract {
   IPyth pyth;
 
+  /**
+   * THIS IS AN EXAMPLE CONTRACT. IT IS NOT INTENDED TO BE USED IN PRODUCTION.
+   * THIS CONTRACT IS FOR DEMONSTRATION PURPOSES ONLY.
+   */
+
+  /**
+      * Instantiate IPyth interface with the address of the Pyth contract.
+      * Pyth contract address can be found at https://docs.pyth.network/price-feeds/contract-addresses/evm.
+
+      * @param pythContract The address of the Pyth contract
+      */
   constructor(address pythContract) {
     pyth = IPyth(pythContract);
   }
 
-  function doSomething(
-    uint someArg,
-    string memory otherArg,
-    bytes[] calldata priceUpdateData
-  ) public payable {
-    // Update the on-chain Pyth price(s)
+  /**
+     * This function is an example of how to interact with the Pyth contract.
+
+     * Fetch the updateData from an off-chain service and pass it to the Pyth contract to update the price feeds.
+     * See https://docs.pyth.network/price-feeds/how-pyth-works/hermes for more information on how to fetch the updateData.
+
+     * @param priceUpdateData The updateData to update the contract with the latest price
+     */
+  function fetchPrice(bytes[] calldata priceUpdateData) public payable {
```

Comment:
same comment as above. Put the documentation on the relevant lines explaining what they do.

And I don't think this should be called `fetchPrice` because it's going to confuse the reader. It should be called something totally unrelated to pyth that's clearly a dummy name. like `yourMethodHere` or something.

Reply:
also can you name this argument `priceUpdate` for consistency with our terminology.